components/breadcrumb/BreadcrumbItem.razor

```diff
@@ -32,25 +31,32 @@
 			@BreadCrumb.Separator
 		</span>
 	}
-
-</span>
+</li>
 
 @code {
+    [Parameter]
+    public RenderFragment ChildContent { get; set; }
 
-	[Parameter]
-	public RenderFragment ChildContent { get; set; }
+    [Parameter]
+    public RenderFragment Overlay { get; set; }
 
-	[Parameter]
-	public RenderFragment Overlay { get; set; }
+    [Parameter]
+    public string Href { get; set; }
 
-	[Parameter]
-	public string Href { get; set; }
+    [Parameter]
+    public EventCallback<Tuple<MouseEventArgs, BreadcrumbItem>> OnClick { get; set; }
 
-	[CascadingParameter]
-	public Breadcrumb BreadCrumb { get; set; }
+    [CascadingParameter]
+    public Breadcrumb BreadCrumb { get; set; }
 
+    private async void _onClick(MouseEventArgs args)
```

Comment:
Ok. I see you changed the signature of the OnClick to just be MouseEventArgs now though. This method isn't needed anymore then since you can directly pass OnClick to the element.